src/components/Search/SelectedRevisionItem.tsx

```diff
@@ -51,6 +50,18 @@ function SelectedRevisionItem({
   const commitMessage = getLatestCommitMessage(item);
   const itemDate = new Date(item.push_timestamp * 1000);
   const repository = repoMap[item.repository_id] ?? 'try';
+  const localDateTime = itemDate
+    .toLocaleString(undefined, {
+      year: '2-digit',
+      month: '2-digit',
+      day: '2-digit',
+      hour: '2-digit',
+      minute: '2-digit',
+      hour12: false,
+      timeZoneName: 'short',
+    })
+    .replace(',', '') // Removes the comma between the date and time
+    .replace(/(\d{2})\/(\d{2})\/(\d{2})/, '$2/$1/$3'); // Reformat date as MM/DD/YY
```

Comment:
This is very fragile, but I understand why you did it.

I need to explain a few things:

The screenshot Carla shared in #767 shows the date and time using the USA notation: MM/DD/YY. But this is confusing to all other users. Indeed in most other parts of the world, we use DD/MM/YY. Therefore for some dates it's confusing. For example 01/10/24 could be either January 10 or October 1st.

My recommendation would be to use `month: "short"` for this reason.

For day, hour, minute, I'd use "numeric" so that they can be 1 digit if possible.

You need to understand that `toLocaleString` with the first argument of `undefined` will return a different format for every language (and even possibly for different browsers and operating systems as well). So using `replace` afterwards won't work always. Instead we need to embrace the fact that the result won't always be exactly the same as specified.

So please remove these `replace` :-)

Finally I think we want to force the `en-US` locale because the interface is in english only currently, so please replace `undefined` with `en-US`.

Reply:
The same comment applies to the other code above.

Reply:
And we also want the tooltip here by the way :-) It could be good to extract this little piece of functionality (showing a date with both the local time + the utc time as a tooltip) to a separate component that we could reuse in these 2 places.